bundles/org.openhab.binding.mybmw/src/main/resources/OH-INF/thing/location-channel-types.xml

```diff
@@ -16,4 +16,9 @@
 		<item-type>String</item-type>
 		<label>Address</label>
 	</channel-type>
+	<channel-type id="home-distance-channel">
+		<item-type>Number:Length</item-type>
+		<label>Distance from Home</label>
+		<state pattern="%d %unit%" readOnly="true"/>
```

Comment:
Consider adding a description also, for example: "Computed distance between vehicle and home location".

Reply:
Indeed. However, after adding the description, the i18n properties file must be regenerated, so it will also include this added description.

Reply:
I don't see it. Perhaps your commit was not pushed?

Reply:
Comitted and pushed 
https://github.com/openhab/openhab-addons/pull/13093/files#diff-f2b38efdede772721841233b944d1008ab78b9f2f198345c6e191d52dfe1eea3

Reply:
See [this comment](https://github.com/openhab/openhab-addons/pull/13093/files#r917091684).